tests/test4_rating_test.py

```diff
@@ -0,0 +1,129 @@
+        return (username, password, projectName, evaluationName, switchGroup, level, checkbox1, checkbox2, checkbox3)
+
+class TestRating(unittest.TestCase):
+    
+    def test_1_SignUp_Existed(self):
+        #If this is not the first time of running this code, then the username would be existed
+        
+        testSignUp = signUp()
+        print("\n\nTesting SignUp\n\n") 
+        
+        (username, password) = configure.configure_test_1_successOrExisted() 
+        (urlCurrent, alertInfo) = testSignUp.Driver_SignUp(username, password)
+        
+        testSignUp.Close()
+
+      
+    
+      
+    def test_3_CreateProject_Success(self):
+        #if first time run, this test will create a project; if not the first time, there won't be duplicate projects created
+        
+        print("\n\nTesting createProject\n\n")  #somehow this is not printed
+        
+
+        (username, password, projectname, projectdescription, studentFile, jsonFile) = configure.configure_test_3_CreateProject_Success()
+        createP = createProject()
+        
+        (urlCurrent, alertInfo)= createP.createProject_attempt(username, password, projectname, projectdescription, studentFile, jsonFile)
+        
+        createP.Close()
+        
+    
+    def test_Evaluations(self):
+        # This evaluation will test for either successfully creating evaluation or fail due to existed evaluation name
+
+        (username, password, projectName, evaluationName) = configure.configure_test_Evaluations()
+        createE = evaluation()
+        
+        (projectURL, alertInfo) = createE.driver_createEvaluation_attempt(username, password, projectName, evaluationName)
+        createE.Close()
+        
+    
+    
+
+
+    def test_Rating_One_Group(self):
+        # Here is only my hardcoding work
+        # due to Out-Of-Index problem, I created a new rubric called "Teamwork1" with one evalution as "Week 1"
+        # in addition, almost all the elements on the page is associated with my userid and the time of the creation of the evaluation
+        
+
+        (username, password, projectName, evaluationName,level, checkbox1, checkbox2, checkbox3) = configure.configure_test_Rating()
+        
+        createR = rating()        
+        (projectURL, metaGroupURL, statusA, statusB, statusC) = createR.driver_Rating_One_Group(username, password, projectName, evaluationName,level, checkbox1, checkbox2, checkbox3)
+        createR.Close()
+        
+        IsProject = projectURL == "http://localhost:5000/load_project/sampleuser13@mailinator.comsampleuser13@mailinator.comTeamwork1full/noAlert"
+        
+        IsMetaGroup = metaGroupURL == "http://localhost:5000/jump_to_evaluation_page/sampleuser13@mailinator.comsampleuser13@mailinator.comTeamwork1full/Week%201/b/***None***/noAlert"
+ 
+        self.assertTrue(IsProject and IsMetaGroup)
+        if checkbox1: self.assertTrue(statusA)
```

Comment:
if checkbox1 was false, would you want to check something?

Reply:
This is no longer apart of test4_rating_test.py